### ISSUE TITLE
The user shall be able to create a new WorkflowTask

### ISSUE THREAD

**Issue by rkpasia** [fractal-analytics-platform/fractal-web#63]
In order to create a new WorkflowTask the user shall be able to specify:

- The task to be associated. 
Selection from a list of tasks fetched from the server. 
Display the name of tasks within the options of select input.
- The `args` property has to be a dynamic input group that enable the user to specify a key and a value associated with that key.
The structure of the object to be created is the following:
`<key:string, value:obj>` where `value` could be `<string | number | <key:string, value:obj>>`.
- `meta` follows the same principles of `args`.

There is another attribute in the payload `order` which will be not supported in the first place.

**Comment by tcompa** [comment]
> There is another attribute in the payload order which will be not supported in the first place.

Let's actually include it (it's an integer attribute).

**Comment by tcompa** [comment]
First version (to be implemented now):

- [ ] As a user, upon adding a WorkflowTask, I can specify a sequence of key-value-type items (where I both choose the key and the value, and say whether type is string/number/boolean).

Example
* `arg1` is a string, and takes value `"value1"`
* `arg2` is a number, and takes values `1`
* `arg3` is a boolean, and takes value `True`

- [ ] The same is done also for "dictionary" types. The web client parses this string and converts it to a dictionary.

E.g. I can say that the argument `arg3` takes value `"{\"a\": \"b\"}"`, and the web client transforms it into the `{"a": "b"}` dictionary.

- [ ] In the tasks package, and then in the DB, we expose JSON schemas for each task's arguments (if it is known to fractal-server). If the schema is there, then the web client exposes those arguments, and only those. If the schema is not available, then the user can add/remove any kind of arguments (as in the previous points).

- [ ] Future feature: We may consider also the possibility of editing a known schema.

- [ ] Future feature: The database already has some default values - which at some point should be pre-populated.

**Comment by tcompa** [comment]
Note: this includes both the `args` and `meta` attributes of the WorkflowTask.

**Comment by tcompa** [comment]
Possible simpler solution:
Adding a WorkflowTask only requires the task and the order. All the argument handling is shifted to the edit functionality.

**Comment by tcompa** [comment]
> Note: this includes both the `args` and `meta` attributes of the WorkflowTask.

@rkpasia We did not discuss this point this morning. I'll clarify later.

TL;DR The `args` JSON attribute is the one where most of the features will be needed, but we also may want to add some (a very few) key-value items in the `meta` JSON attribute.
